src/encoded/schemas/changelogs/file.md

```diff
@@ -12,6 +12,7 @@
 * Added *origin_batches* calculated property
 * Added *regulatory elements prediction model* to the enum list for *output_type*
 * Added *genome subcompartments* and *chromatin stripes* to the enum list for *output_type*
+* Added *TF binding prediction model*, *enhancer prediction model* and *"*promoter prediction model*  to the enum list for *output_type*, restricted the use of *regulatory elements prediction model* to DCC personell only
```

Comment:
Just one typo - `personell` should be `personnel`

Reply:
Also you have a double quote and extra asterisk in `*"*promoter prediction model*` that should be removed to be `*promoter prediction model*`

Reply:
fixed